meson.build

```diff
@@ -335,6 +335,13 @@ unsigned int foobar (const struct test *p) { return p->part; }
 '''
 have_bitfields_in_packed_structs = meson.get_compiler('c').compiles(bitfields_in_packed_structs_code, name : 'packed structs may contain bitfields')
 
+# CompCert does not have __builtin_mul_overflow
+builtin_mul_overflow_code = '''
+#include <stddef.h>
+int overflows (size_t a, size_t b) { size_t x; return __builtin_mul_overflow(a, b, &x); }
```

Comment:
Hrm. I'm not sure how this test could ever fail -- calling an undeclared function is legal in C. For example, this file compiles successfully using GCC, although it does emit a warning:

#include <stddef.h>

int overflows(size_t a, size_t b, size_t *x)
{
	return __builtin_mul_notdefined(a, b, x);
}

Maybe adding '-Werror' to the compiler flags would cause this to generate an error instead?